libs/sdk-backend-tiger/src/backend/workspace/catalog/factory.ts

```diff
@@ -97,10 +97,10 @@ export class TigerWorkspaceCatalogFactory implements IWorkspaceCatalogFactory {
 
     private getCatalogItemId = (item: CatalogItem) => {
```

Comment:
Should we change the name to something like `getCatalogItemSortingKey`? Now that it no longer returns ids and is only used for sorting?